testnets/manifest/assetlist.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "manifest",
+  "assets": [
+    {
+      "description": "Manifest testnet native token",
+      "denom_units": [
+        {
+          "denom": "umfx",
+          "exponent": 0
+        },
+        {
+          "denom": "mfx",
+          "exponent": 6
+        }
+      ],
+      "base": "umfx",
+      "name": "Manifest Testnet Token",
+      "display": "mfx",
+      "symbol": "MFX",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/manifest/images/manifest.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/manifest/images/manifest.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/manifest/images/manifest.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/manifest/images/manifest.svg"
+        }
+      ]
+    },
+    {
+      "description": "Proof of Authority token for the Manifest testnet",
+      "denom_units": [
+        {
+          "denom": "upoa",
+          "exponent": 0
+        },
+        {
+          "denom": "poa",
+          "exponent": 6
+        }
+      ],
+      "base": "upoa",
+      "name": "Manifest Testnet Token",
+      "display": "poa",
+      "symbol": "POA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/manifest/images/manifest.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/manifest/images/manifest.svg"
+      },
+      "images": [
```

Comment:
Please only upload image files with a perfectly square canvas. The current images are slightly taller than they are wide (hence, recangular).